README.md

```diff
@@ -108,19 +108,26 @@ Add another argument to the local upgrade script:
 Once you are running myNode, you can easily update the software yourself!
 
 1. Start by running myNode on your device via the instructions above in "Running myNode"
-2. Clone the latest release from the git repo on your PC or laptop
+2. Clone the current myNode Github repo onto your machine (this is easiest if you do it on the myNode machine using the machine's native terminal. For VM version, minimize the initial myNode window, right click on desktop background, and select Terminal emulator)
     * Run `git clone https://github.com/mynodebtc/mynode.git`
+3. Change directory into the cloned repo
     * Run `cd mynode`
+4. Switch to the latest release version branch
     * Run `git checkout tags/latest_release`
```

Comment:
this tag doesn't refer to the latest version, I believe it was created erroneously

Reply:
That's the original instruction. I hadn't changed it at all. Also, that's the command I've used to upgrade myNode on VM and it does upgrade to the latest version. However, if it needs to be changed, that's not my purview. I was just adding information to make the instructions better, but not change any of them materially.

Reply:
my mistake